.github/workflows/test-ci-reusable.yml

```diff
@@ -35,14 +35,14 @@ jobs:
   controller-test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
       with:
         repository: ${{ inputs.repository }}
         ref: ${{ inputs.ref }}
-    - uses: actions/setup-go@v2
+    - uses: actions/setup-go@v4
       with:
         go-version: '1.20.*'
-    - uses: actions/cache@v2
+    - uses: actions/cache@v4
```

Comment:
pretty sure the latest version of actions/cache is v3

Reply:
yes: actions/setup-go and actions/checkout only are v4